update.sh

```diff
@@ -17,8 +49,6 @@ fi
 # TODO: Should be able to specify target architecture manually
 arch=$(get_arch)
 
-yarnVersion="$(curl -sSL --compressed https://yarnpkg.com/latest-version)"
```

Comment:
Keep this line and change it to

`latestYarnVersion="$(curl -sSL --compressed https://yarnpkg.com/latest-version)"`

Reply:
Putting it at the top means that you're querying  https://yarnpkg.com/ via curl even in situations where you don't need to (like skipping the yarn update. But..then we query it multiple times during repeated calls to `update_node_version`.

Maybe put the `latestYarnVersion` logic in an if block and only run it if `${SKIP}` evals to false?

Reply:
Oh, good call!

Reply:
I pushed another change to set yarnVersion globally if `SKIP = false`

Reply:
And another change to set the alpine version globally when `SKIP = false`